tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -220,97 +220,95 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
-             |
-             |/b/others/Script.sc
-             | // scala $scalaVersion
-             |val a = ""
-             |
-             |/b/notThis.sc
-             | // scala $scalaVersion
-             |val a = ""
-             |
-             |""".stripMargin
-        )
-        _ <- server.didOpen("b/otherMain.sc")
-        _ <- server.didOpen("b/other.sc")
-        _ <- server.didOpen("b/otherScript.sc")
-        _ <- server.didOpen("b/others/Script.sc")
-        _ <- server.didOpen("b/notThis.sc")
-        _ <- server.executeCommand(ServerCommands.StartAmmoniteBuildServer)
-        _ <- server.didSave("b/otherMain.sc")(identity)
-        _ <- server.didSave("b/other.sc")(identity)
-        _ <- server.didSave("b/otherScript.sc")(identity)
-        _ <- server.didSave("b/others/Script.sc")(identity)
-        _ <- server.didSave("b/notThis.sc")(identity)
-        expectedCompletionList = """|other.sc
-                                    |otherScript.sc
-                                    |others""".stripMargin
-        completionList <- server.completion(
-          "b/otherMain.sc",
-          "import $file.other@@",
-        )
-        _ = assertNoDiff(completionList, expectedCompletionList)
-
-      } yield ()
-    }
+  test("file-completion") {
+    for {
+      _ <- initialize(
+        s"""
+           |/metals.json
+           |{
+           |  "a": {
+           |    "scalaVersion": "$scalaVersion"
+           |  }
+           |}
+           |/b/otherMain.sc
+           | // scala $scalaVersion
+           |import $$file.other
+           |
+           |/b/otherScript.sc
+           | // scala $scalaVersion
+           |val a = ""
+           |           
+           |/b/other.sc
+           | // scala $scalaVersion
+           |val a = ""
+           |
+           |/b/others/Script.sc
+           | // scala $scalaVersion
+           |val a = ""
+           |
+           |/b/notThis.sc
+           | // scala $scalaVersion
+           |val a = ""
+           |
+           |""".stripMargin
+      )
+      _ <- server.didOpen("b/otherMain.sc")
+      _ <- server.didOpen("b/other.sc")
+      _ <- server.didOpen("b/otherScript.sc")
+      _ <- server.didOpen("b/others/Script.sc")
+      _ <- server.didOpen("b/notThis.sc")
+      _ <- server.executeCommand(ServerCommands.StartAmmoniteBuildServer)
+      _ <- server.didSave("b/otherMain.sc")(identity)
+      _ <- server.didSave("b/other.sc")(identity)
+      _ <- server.didSave("b/otherScript.sc")(identity)
+      _ <- server.didSave("b/others/Script.sc")(identity)
+      _ <- server.didSave("b/notThis.sc")(identity)
+      expectedCompletionList = """|other.sc
+                                  |otherScript.sc
+                                  |others""".stripMargin
+      completionList <- server.completion(
+        "b/otherMain.sc",
+        "import $file.other@@",
+      )
+      _ = assertNoDiff(completionList, expectedCompletionList)
 
-  if (scalaVersion != V.ammonite3)
```

Comment:
Note for the reviewers: the body of the test is not changed at all. I only removed the condition for excluding Scala 3